src/ucp/stream/stream_recv.c

```diff
@@ -213,8 +218,9 @@ ucp_stream_process_rdesc(ucp_recv_desc_t *rdesc, ucp_ep_ext_proto_t *ep_ext,
 }
 
 static UCS_F_ALWAYS_INLINE void
-ucp_stream_recv_request_init(ucp_request_t *req, void *buffer, size_t count,
-                             size_t length, ucp_datatype_t datatype,
+ucp_stream_recv_request_init(ucp_ep_h ep, ucp_request_t *req, void *buffer,
```

Comment:
Should the function signature similar to other request init functions? Like these two:

https://github.com/openucx/ucx/blob/master/src/ucp/tag/tag_send.c#L119
https://github.com/openucx/ucx/blob/master/src/ucp/stream/stream_send.c#L27

Request is the first parameter, then ep in both the above cases.